src/app/src/main/resources/config.properties

```diff
@@ -30,5 +30,6 @@ oauth2_client_id=fakeId
 oauth2_client_secret=fakeSecret
 oauth2_client_url=fakeUrl
 oauth2_client_audience=naomi
-oauth2_client_adr_url=http://localhost:5000
-oauth2_login_method=false
\ No newline at end of file
+oauth2_client_adr_server_url=http://localhost:5000
```

Comment:
Is this not going to be the same as the adr_url in practice. or at least share the same base url? Why do we need to keep this separately?
UPDATE: Thanks for the local test config, looks like they are using an unrelated base url!
I think it would be good to include all of the config for testing here, apart from the secret. which should be stored and fetched from the vault.